a3p-integration/proposals/n:upgrade-next/test/terminate-governed-instance.test.js

```diff
@@ -0,0 +1,33 @@
+/* eslint-env node */
+import test from 'ava';
+import '@endo/init/debug.js';
+
+import { retryUntilCondition } from '@agoric/client-utils';
+import { evalBundles } from '@agoric/synthetic-chain';
+import { arrayIsLike } from '@agoric/internal/tools/ava-assertions.js';
+import { getGovernedVatDetails } from '../test-lib/vats.js';
+
+test('verify governed vat termination', async t => {
+  const getVats = () => getGovernedVatDetails('-ATOM-USD_price_feed', true);
+
+  const initialVats = await getVats();
+  t.log('initial instances', ...initialVats);
+  arrayIsLike(t, initialVats, [
+    { nameSuffix: '-governor', terminated: false },
+    { nameSuffix: '', terminated: false },
+  ]);
+
+  // generated by "sdk-generate" in ../package.json
+  await evalBundles('generated/terminate-price-feed');
+
+  const checkForTermination = vats => {
+    t.log('remaining instances', ...vats);
+    return vats.length === 0;
```

Comment:
this looks like we're terminating _all_ the ATOM-USD price feed instances.

am I reading it right? help?

_grr... I typed this comment in once already but the github UI threw it away somehow._

Reply:
Am I lost? I thought the context here is:
 1. original ATOM-USDC price feed starts with the launch of vaults
 2. we deploy new price feeds, leaving the old one running
 3. we stop the old ATOM-USDC price feed instance but leave the new one running

and this coreEval is step 3. We don't want to kill the instance from 2.

Reply:
Updated; your understanding was correct and mine was not.